.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: flake8
         working-directory: ./scripts
-        run: flake8 ./*.py --exclude oligomapOutputToSam_nhfiltered.py
+        run: flake8 ./*.py
```

Comment:
Why include `./*.py` - by default `flake8` only checks `.py` files, no?

Reply:
Ah, right, I forgot that the `scripts/` directory _also_ has a `tests/` directory.

But yeah, probably good to have at least some linting on the tests as well, so yeah, maybe open another issue.